src/amo/components/Search/index.js

```diff
@@ -152,6 +165,7 @@ export function mapStateToProps(state) {
 }
 
 export default compose(
+  translate({ withRef: true }),
```

Comment:
I don't think you'll actually need the `withRef: true` here; we use it elsewhere but I think that was for an old testing thing or something? Try leaving it out and if it still works it's fine.